src/wp-includes/formatting.php

```diff
@@ -5935,7 +5935,7 @@ function _print_emoji_detection_script() {
 		 *
 		 * @param string $url The emoji base URL for png images.
 		 */
-		'baseUrl' => apply_filters( 'emoji_url', 'https://s.w.org/images/core/emoji/16.0.1/72x72/' ),
+		'baseUrl' => apply_filters( 'emoji_url', 'https://s.w.org/images/core/emoji/17.0.1-1/72x72/' ),
```

Comment:
```suggestion
		'baseUrl' => apply_filters( 'emoji_url', 'https://s.w.org/images/core/emoji/17.0.1/72x72/' ),
```

Why `-1`?

Reply:
While testing this, I accidentally requested the asset _before_ it existed on the w.org CDN. This results in a cached 404 request. Since this is cached indefinitely, the folder was renamed so that all URLs are uncached.

I just [updated the meta ticket](https://meta.trac.wordpress.org/ticket/8122#comment:2) with this information as well.